config/routes.rb

```diff
@@ -2,6 +2,15 @@
 require 'sidekiq/cron/web'
 
 Rails.application.routes.draw do
+  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+    # Protect against timing attacks:
+    # - See https://codahale.com/a-lesson-in-timing-attacks/
+    # - See https://thisdata.com/blog/timing-attacks-against-string-comparison/
+    # - Use & (do not use &&) so that it doesn't short circuit.
+    # - Use digests to stop length information leaking (see also ActiveSupport::SecurityUtils.variable_size_secure_compare)
```

Comment:
It's maybe a bit iffy to refer to `variable_size_secure_compare` given it hasn't existed since Rails 5, so isn't currently available in the implementation of ActiveSupport::SecurityUtils used in this codebase.